docs/sources/developers/plugins/annotations.md

```diff
@@ -0,0 +1,137 @@
++++
+title = "Add annotations to your data source plugin"
+type = "docs"
++++
+
+# Add annotations to your data source plugin
+
+This guide explains how to add support [annotations]({{< relref "reference/annotations.md" >}}) to your data source plugin.
+
+This guide assumes that you're already familiar with how to [Build a data source plugin]({{< relref "../../../../../tutorials/build-a-data-source-plugin.md" >}}).
+
+1. Let Grafana know that your data source plugin can return annotations events, by adding `"annotations": true` to the [plugin.json]({{< relref "metadata.md" >}}) file.
```

Comment:
We should put an h2 at the start of this process to hold the procedure. What I often do is use the more detailed task name for this (like what you have for your topic title) and then use a more general title for the topic.

Title: Add annotation support to plugins OR Annotation support for plugins
Task: Add support for annotations to your data source plugin